lib/infobox/node.js

```diff
@@ -61,9 +61,9 @@ define(['sorttable', 'snabbdom', 'd3-interpolate', 'moment', 'helper', 'utils/no
       }
 
       function renderNeighbourRow(n) {
-        var icons = '';
+        var icons = [V.h('span', { props: { className: 'icon ion-' + (n.link.type === 'wifi' ? 'wifi' : 'share-alt'), title: _.t(n.link.type) } })];
```

Comment:
Maybe usw `link.indexOf('wifi') === 0` for later implementation of different wifi types like `wifi24` and `wifi5`.